showcase/app/templates/components/form/base-elements.hbs

```diff
@@ -21,32 +21,32 @@
     <SF.Item @label="With optional indicator">
       <Hds::Form::Label @isOptional={{true}}>This is the label</Hds::Form::Label>
     </SF.Item>
-    <SF.Item as |SF|>
-      <SF.Label>With structured content (eg. a <code>flex</code> layout and a <code>&lt;Badge&gt;</code>)</SF.Label>
+    <SF.Item as |SFI|>
+      <SFI.Label>With structured content (eg. a <code>flex</code> layout and a <code>&lt;Badge&gt;</code>)</SFI.Label>
       <Hds::Form::Label>
         <div class="shw-component-form-base-elements-container-with-badge">
           This is the label
           <Hds::Badge @size="small" @text="Some badge" @color="highlight" />
         </div>
       </Hds::Form::Label>
     </SF.Item>
-    <SF.Item>
-      <SF.Label>With structured content (eg. a
+    <SF.Item as |SFI|>
+      <SFI.Label>With structured content (eg. a
         <code>flex</code>
         layout and a
-        <code>&lt;Badge&gt;</code>) and required indicator</SF.Label>
+        <code>&lt;Badge&gt;</code>) and required indicator</SFI.Label>
       <Hds::Form::Label @isRequired={{true}}>
         <div class="shw-component-form-base-elements-container-with-badge">
           This is the label
           <Hds::Badge @size="small" @text="Some badge" @color="highlight" />
         </div>
       </Hds::Form::Label>
     </SF.Item>
-    <SF.Item>
```

Comment:
This was not exporting the `as |SF|` so the label was not rendered (now it's rendered, and Percy picked up the difference)